app/models/event.rb

```diff
@@ -3,6 +3,7 @@
 # Physical event at a location
 class Event < ApplicationRecord
   alias_attribute :id, :event_id
+  attribute :estimated_distance, :float, default: -> { 0.0 }
```

Comment:
I don't think we need this as an attribute. The method is enough.

Reply:
At some point I got an error because it wasn't defined.  I could take it out and see what happens.

Reply:
I was able to remove this.